aws-sdk-rails.gemspec

```diff
@@ -24,12 +24,20 @@ Gem::Specification.new do |spec|
   spec.add_dependency('aws-sdk-ses', '~> 1', '>= 1.50.0') # for ActionMailer
   spec.add_dependency('aws-sdk-sesv2', '~> 1', '>= 1.34.0') # for ActionMailer
   spec.add_dependency('aws-sdk-sqs', '~> 1', '>= 1.56.0') # for ActiveJob
+  spec.add_dependency('aws-sdk-sns', '~> 1.75') # for ActionMailbox
 
   spec.add_dependency('aws-sessionstore-dynamodb', '~> 2') # includes DynamoDB
   spec.add_dependency('concurrent-ruby', '>= 1.3.1') # Utilities for concurrent processing
-  spec.add_dependency('railties', '>= 5.2.0') # encrypted credentials
+  spec.add_dependency('railties', '>= 7.0.0') # encrypted credentials
+  spec.add_dependency('actionmailbox', '>= 7.0.0')
+
+  spec.add_runtime_dependency('aws-sdk-s3', '~> 1.152')
```

Comment:
I'm not sure if we should just add it is a dependency? Or if SNS needs to be runtime too

Reply:
We can add it as a runtime dependency here.

Reply:
Actually...no, I don't think that can work due to the initializer and mounting

Reply:
I mean, we can make it a regular dependency no problem